locus/src/main/java/com/birjuvachhani/locus/LocusActivity.kt

```diff
@@ -66,10 +74,10 @@ class LocusActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRe
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_location_permission)
-
+        config = Configuration(getLocusLocationRequest(getAvailableService()))
```

Comment:
No need to initialize config here as it should parsed from the Intent!

Reply:
I have removed this line already.